pom.xml

```diff
@@ -23,7 +23,7 @@
     <name>CouchDB Lucene</name>
     <description>Full-text indexing for CouchDB</description>
     <url>http://github.com/rnewson/couchdb-lucene/</url>
-    <version>2.1.0-SNAPSHOT</version>
+    <version>2.2.0-SNAPSHOT</version>
```

Comment:
revert this hunk, the feature enhancement doesn't mean we'll release as 2.2.0 and skip over 2.1.0. :)